### DISCUSSION TITLE
Click detectable

### DISCUSSION THREAD

**Discussion by skyslovt** [ultrafunkamsterdam/undetected-chromedriver#1730] [Q&A]
I think that selenium clicks are detectable for hcaptcha, because when I solve it manually I get success on the page but when I use selenium clicks I don't, is there any way to make these clicks more "humanized"? I'm currently using this to click
`actions.move_to_element_with_offset(element, random.uniform(-5, 5), random.uniform(-5, 5)).click().perform()`

**Comment by GGontijo** [comment]
Did you tried selecting with a keyboard command like 'space' or 'enter' with something like pyautogui? hcaptcha is very difficult to bypass...
